packages/react/src/solana/hooks/program/useProgram.ts

```diff
@@ -92,13 +99,15 @@ export function programQuery<TProgramType extends ProgramType>(
  *
  * @public
  */
-export function useProgram<TProgramType extends ProgramType>(
-  address: RequiredParam<string>,
-  type?: TProgramType,
-) {
+export function useProgram<
+  TProgram extends ValidProgram,
+  TProgramType extends ProgramType | undefined = undefined,
```

Comment:
hmmmm any way to not have the ProgramType generic?

Reply:
why?